### ISSUE TITLE
Can't run Magnetar CLI

### ISSUE THREAD

**Issue by Ovi05** [Kyliekyler/MAGNETAR#23]
Hello! I'm trying to configure Magnetar in Termux, but when I run the command, I get the following error:
`/system/bin/sh: magnetar: inaccessible or not found`. The module appears as installed and running in KernelSU (version 11928). There's no notification, but it seems to be doing something, since if automatically changed the governor to schedutil after installation. Phone model is Xiaomi 12 Pro.

**Comment by Kyliekyler** [comment]
> Hello! I'm trying to configure Magnetar in Termux, but when I run the command, I get the following error: `/system/bin/sh: magnetar: inaccessible or not found`. The module appears as installed and running in KernelSU (version 11928). There's no notification, but it seems to be doing something, since if automatically changed the governor to schedutil after installation. Phone model is Xiaomi 12 Pro.

Perhaps you can try running `su -c magnetar`

**Comment by Ovi05** [comment]
> Perhaps you can try running `su -c magnetar`

It gives me the same error

**Comment by Kyliekyler** [comment]
> > Perhaps you can try running `su -c magnetar`
> 
> It gives me the same error

Please provide installation logs

**Comment by Ovi05** [comment]
> Please provide installation logs

Where can I find them?

**Comment by Kyliekyler** [comment]
> > Please provide installation logs
> 
> Where can I find them?

Install magnetar again and save the logs from the root manager you used to install magnetar

**Comment by Ovi05** [comment]
> Install magnetar again and save the logs from the root manager you used to install magnetar

Here they are:
[KernelSU_install_log_2024-08-04-22-47-45.log](https://github.com/user-attachments/files/16488721/KernelSU_install_log_2024-08-04-22-47-45.log)

**Comment by Kyliekyler** [comment]
> > Install magnetar again and save the logs from the root manager you used to install magnetar
> 
> Here they are: [KernelSU_install_log_2024-08-04-22-47-45.log](https://github.com/user-attachments/files/16488721/KernelSU_install_log_2024-08-04-22-47-45.log)

Seems to be that it got installed correctly. May I ask what terminal emulator you're using to execute magnetar?

**Comment by Ovi05** [comment]
> Seems to be that it got installed correctly. May I ask what terminal emulator you're using to execute magnetar

I'm using termux, but I aldo tried 2 other terminal emulator apps, with the same result

**Comment by Kyliekyler** [comment]
> > Seems to be that it got installed correctly. May I ask what terminal emulator you're using to execute magnetar
> 
> I'm using termux, but I aldo tried 2 other terminal emulator apps, with the same result

Can you tell how to exactly recreate the issue? I was not able to get the same error on my end. If possible, please do include screenshots, thanks

**Comment by Ovi05** [comment]
> Can you tell how to exactly recreate the issue? I was not able to get the same error on my end. If possible, please do include screenshots, thanks

Sorry for taking so long to respond (I was quite busy lately). I noticed that under su no command works. Here's nano as an example:
![IMG_20240808_002129](https://github.com/user-attachments/assets/944e4f03-9272-4000-b7b6-8d8e612a673c)
However I installed tsu and most commands seem to work as root now (nano, for example, stars as expected). However, running magnetar still gives an error, this time `magnetar: command not found`
![IMG_20240808_002408](https://github.com/user-attachments/assets/0719cc0b-a96e-4f7b-8455-6dbdb0b688df)

So, overall, the step by step is:
- Install magnetar in kernelsu
- Reboot
- Open termux
- Run `pkg install tsu`
- Run `tsu`, followed by `magnetar`
